packages/rum-core/src/domain/rumEventsCollection/error/errorCollection.ts

```diff
@@ -105,14 +105,15 @@ function processError(
     },
     type: RumEventType.ERROR as const,
   }
+
   const inForeground = foregroundContexts.isInForegroundAt(error.startClocks.relative)
   if (inForeground) {
     rawRumEvent.view = { in_foreground: inForeground }
   }
 
-  const viewContext = viewContexts.findView(error.startClocks.relative)
-  if (viewContext && !isEmptyObject(viewContext.featureFlagEvaluations)) {
-    rawRumEvent.feature_flags = viewContext.featureFlagEvaluations
+  const featureFlagContext = featureFlagContexts.findFeatureFlagEvaluations(error.startClocks.relative)
+  if (featureFlagContext && !isEmptyObject(featureFlagContext)) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what’s best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**
You probably do not need to check `!isEmptyObject(featureFlagContext)`

Reply:
The feature flag context with an empty object is created for each `VIEW_CREATED`. If no they are no calls to `addFeatureFlagEvaluation`, we don't want to see the attribute in Datadog. Same logic applied for `customTimings` in `viewCollections`. Wdyt?